tests/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "require": {
+    },
+    "require-dev": {
+        "satooshi/php-coveralls": "^1.0",
+        "phpunit/phpunit": "^5.6"
```

Comment:
We might want to add a note in the readme specifying that this requires PHP 5.6 or higher.